{"name": "test_multiple_login_page_success[invalid_user4-wrong_password]", "status": "passed", "parameters": [{"name": "username", "value": "'invalid_user4'"}, {"name": "password", "value": "'wrong_password'"}], "start": 1753503735233, "stop": 1753503736977, "uuid": "75d30004-f6b9-4dc9-9344-48ec6d06e7c0", "historyId": "8169f900955b8e47d48e3638b02b295f", "testCaseId": "90d4f72dd78db4ae114ce1f1cc9587a0", "fullName": "tests.test_multiple_login#test_multiple_login_page_success", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_multiple_login"}, {"name": "host", "value": "ahmads-MacBook-Pro.local"}, {"name": "thread", "value": "60587-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_multiple_login"}], "titlePath": ["tests", "test_multiple_login.py"]}